README.md

```diff
@@ -31,23 +31,27 @@ Use the sbt plugin for Pekko Paradox:
 addSbtPlugin("org.apache.pekko" % "pekko-sbt-paradox" % "<version>")
 ```
 
-### Usage within JDK 1.8 projects
+## Usage
```

Comment:
There are 2  ##`Usage` titles in the readme (this line and line 27). Is it possible to merge the sections?

Reply:
Woops, thanks for noticing. Just rebased PR with the duplicate removed